File: VB/ReplaceDefaultActionInListView.Module.Win/WinModule.vb
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports System.ComponentModel

Imports DevExpress.ExpressApp

Namespace ReplaceDefaultActionInListView.Module.Win
    <ToolboxItemFilter("Xaf.Platform.Win")> _
    Public NotInheritable Partial Class ReplaceDefaultActionInListViewWindowsFormsModule
        Inherits ModuleBase

        Public Sub New()
            ModelDifferenceResourceName = "ReplaceDefaultActionInListView.Module.Win.Model.DesignedDiffs"
            InitializeComponent()
        End Sub
    End Class
End Namespace

File: CS/ReplaceDefaultActionInListView.Module.Win/WinModule.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.ComponentModel;

using DevExpress.ExpressApp;

namespace ReplaceDefaultActionInListView.Module.Win {
    [ToolboxItemFilter("Xaf.Platform.Win")]
    public sealed partial class ReplaceDefaultActionInListViewWindowsFormsModule : ModuleBase {
        public ReplaceDefaultActionInListViewWindowsFormsModule() {
            ModelDifferenceResourceName = "ReplaceDefaultActionInListView.Module.Win.Model.DesignedDiffs";
            InitializeComponent();
        }
    }
}
